### PR TITLE
add a file for my town

### PR DESCRIPTION
**Pull request by joshuadruley**
The purpose of this pull request is to add a hometown file for my town :bridge_at_night: 

Some :question: that I want you all to help me answer:
1. If you have been to my town, what did you do? What did you think of it?
1. If you have kids, what kinds of things do they do at night?  Sleep?
1. What kind of food do you like?

### CODE REVIEW

**Comment by andrew-oberstar** [comment]
1. I've been to town before. It's great!
1. No kids.
1. Many kinds of food. :hamburger: :pizza: :spaghetti: :hotdog:

File: joshuadruley-town.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Town
+
+### Restaurants
+- McD's
```
Comment (andrew-oberstar) [inline joshuadruley-town.md]: Should we trademark this?

File: joshuadruley-town.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Town
+
+### Restaurants
+- McD's
+
+### Places the locals like
+- Key's Cafe
+
+### Places tourists like
+- Mall of America
```
Comment (andrew-oberstar) [inline joshuadruley-town.md]: Maybe this too?

**Review by andrew-oberstar — CHANGES_REQUESTED** [review]
Don't want to get sued.

**Review by andrew-oberstar — APPROVED** [review]

**Comment by joshuadruley** [comment]
This Pull Request resolves #41

**Comment by andrew-oberstar** [comment]
:shipit:  :+1:

**Pull request merged**